### ISSUE TITLE
Admin: Identify next set of metrics for visualization on analytics board

### ISSUE THREAD

**Issue by vickyszuchin** [cisagov/manage.get.gov#2044]
### Issue description

Identify next set of metrics for visualization on analytics board per Product needs. 

### Acceptance criteria

- [ ] Add/Update key metrics spreadsheet per discussion with Product
- [ ] Select the metrics that hold priority for visualization on the analytic board (mark as Priority 2)
- [ ] Create dev  tickets for implementation

### Additional context

[Key metrics spreadsheet](https://docs.google.com/spreadsheets/d/1jEI2Vuy76HnRQoFnV5HbFrKrvlsbU07iWz7wTt2RZDQ/edit#gid=0)
[Product huddle notes](https://docs.google.com/document/d/1jatolB9YaQLITwqx25y0B1u5J23S_J19PawXtIaov8o/edit) on 4/17/2024

### Links to other issues

Related to: #1320 and #1789

**Comment by abroddrick** [comment]
@vickyszuchin I don't think this should be a discovery ticket really, I think we should just have a meeting to go over what is listed really quick ( or just use the refinement meeting) and then pick the next item to be done. The one ticket idea was just so the one ticket could be broken into multiple when we are in refinement and we can determine priority then.

**Comment by katypies** [comment]
@PaulKuykendall is this still relevant? Should it be sprinted with an actionable deadline?

**Comment by abroddrick** [comment]
@katypies this is the one I brought up that we were supposed to discuss at a fireside and never did. This ticket was meant to be used to break into key items that we want for the analytics board going forward

**Comment by witha-c** [comment]
@katypies and @h-m-f-t This refined ticket replaces the new ticket I mentioned creating to capture metrics/analytics redesign.
